### PR TITLE
Remove dependency to Terragrunt + Add AWS SSM config debug logs

### PR DESCRIPTION
**Pull request by julienduchesne**
**Removed dependency to Terragrunt libs**
The size is just 37MB instead of 40MB while like 90% of the dependencies went away. The go compiler seems to be very good at removing unneeded code
Check out this commit: https://github.com/coveooss/tgf/pull/149/commits/c195bd7aa807b06fd6f8fb173bec1c9e9794e82c

**Added new debug logs to the AWS SSM Config method**
Users could not easily figure out what was happening without logs
Also added a TGF_USE_AWS_CONFIG env variable in case we want to force TGF to fetch its config from SSM. This is useful in the case of an EC2 instance that doesn't have the AWS CLI installed. All it has is the 169.254.169.254 endpoint
Check out this commit: https://github.com/coveooss/tgf/pull/149/commits/7d4f4d6bf857cfe3240dd58cbd2fde8719401806

### CODE REVIEW

**Comment by julienduchesne** [comment]
The size is just 37MB instead of 40MB while like 90% of the dependencies went away. The go compiler seems to be very good at removing unneeded code

**Comment by julienduchesne** [comment]
Also made the tests run on windows, mac and linux. It's free (open source)

File: README.md [hunk 1]
```diff
@@ -218,7 +220,7 @@ Flags:
       --temp                    ON by default: Map the temp folder to a local folder, use --no-temp to disable
       --temp-location=TEMP-LOCATION  
                                 Determine where the temporary work folder 'tgf' inside the docker image is mounted:
-                                
+
```
Comment (jocgir) [inline README.md:223]: I wondered why you changed this line. Then, then I realized that kingpin inserted an unwanted line here.

**Review by jocgir — APPROVED** [review]

**Review by ycanty — APPROVED** [review]
Nice!  This takes care of removing the terragrunt module dependencies.  I think we should make another PR to remove other terragrunt dependencies. `docker.go` still has references to terragrunt: it has logic based on the terragrunt entry point, and looks for some TERRAGRUNT* environment variables.  Also, some command line options are terragrunt-specific, and the Makefile uses a terragrunt script.

**Pull request merged**